### PR TITLE
Update guidelines to reflect the new Graphistry version

### PR DESCRIPTION
**Pull request by aucahuasi**
- Update CUDA 11.5 to CUDA 11.8
- Update ubuntu 20.04 to ubuntu 22.04
- Update RAPIDS version (23.10) and new docker conventions (conda env).
- Update information about Azure Marketplace and NVIDIA GPUDirect Storage (GDS).

### CODE REVIEW

**Review by DataBoyTX — CHANGES_REQUESTED** [review]
I think there's one file for the setup scripts that you might have overlooked: https://github.com/graphistry/graphistry-cli/blob/a6fcc76f5af420903d4d406e06f97aca5ce0235a/docs/ubuntu_20_04_setup.sh#L299

but other than that, LGTM!

**Comment by aucahuasi** [comment]
> 11.0.3

Good catch, thanks Thomas! Solved!

**Review by lmeyerov — APPROVED** [review]

**Comment by lmeyerov** [comment]
Looks good - let's merge when the new release goes out

File: docs/azure_marketplace.md [hunk 1]
```diff
@@ -137,8 +137,23 @@ graphistry@d4afa8b7ced5:~$ go version
...
-### 8. Marketplace FAQ
+**Temporary Workaround**
+To ensure stability and performance of Graphistry on Azure, GDS support has been disabled by default. This adjustment has been applied to all Azure Marketplace images by setting the environment variable `LIBCUDF_CUFILE_POLICY` to `OFF`.  See the official Magnum IO GPUDirect Storage (GDS) documentation here for more information:
+https://docs.rapids.ai/api/cudf/nightly/user_guide/io/io/#magnum-io-gpudirect-storage-integration
+
+**Future Considerations**
+Monitoring of this issue is ongoing to identify a permanent solution.  Once Azure resolves this issue, GDS support will be re-enabled to take full advantage of its performance benefits.  Customers will be informed of any updates or changes regarding this matter.
+
+**Recommendations**
+Users may override the default setting and enable GDS support manually by setting the environment variable in the `data/config/custom.env` file (a Docker Compose environment file). For example: `LIBCUDF_CUFILE_POLICY=ALWAYS`.
```
Comment (lmeyerov) [inline docs/azure_marketplace.md:152]: also add to `azure.md` that users most explicitly set `LIBCUDF_CUFILE_POLICY=OFF` as well, e.g., would happen if someone uses the tarball or AKS

**Pull request merged**